rfc-095-long-term-future-apps.md

```diff
@@ -0,0 +1,90 @@
+| feedback | Serves contact pages on GOV.UK | Rename to "feedback-frontend" |
+| finder-frontend | Serves search pages for GOV.UK | Rename to "search-frontend". |
+| frontend | Serves the homepage, transactions and some index pages on GOV.UK | Rename to "custom frontend" |
+| government-frontend | Serves government pages on GOV.UK | Rename to "content-frontend" |
+| maslow | Create and manage needs on GOV.UK | Rename to "user-needs-publisher" |
+| rummager | Search API for GOV.UK | Rename to "search-api" |
+| smart-answers | Serves smart answers on GOV.UK | Rename to "smart-answers-frontend" |
+| support | Forms to raise Zendesk tickets to be used by Government personnel on GOV.UK | Rename to "support-admin" |
+| transition | Managing redirects for sites moving to GOV.UK. | Rename to "transition-admin" |
+
+[cg]: https://github.com/alphagov/govuk_publishing_components
+
+### No changes expected
+
+| Name | Description | Rough plan |
+| -- | -- | -- |
+| asset-manager | Manages uploaded assets (images, PDFs etc.) for applications on GOV.UK | No change expected |
+| authenticating-proxy | Allows authorised users to access the GOV.UK draft stack | No change expected |
+| cache-clearing-service | Clears various caches when new content is published. | No change expected |
+| contacts-admin | Publishes HMRC contact information on GOV.UK | No change expected |
+| content-data-admin | A front end for the data warehouse | No change expected |
+| content-store | API for content on GOV.UK | No change expected |
+| content-tagger | Tool to tag content and manage the taxonomy on GOV.UK | No change expected |
+| email-alert-api | Sends email alerts to the public for GOV.UK | No change expected |
+| email-alert-frontend | Serves email alert signup pages on GOV.UK | No change expected |
+| hmrc-manuals-api | API for HMRC to publish manuals to GOV.UK | No change expected |
+| imminence | Find My Nearest API and management tools on GOV.UK | No change expected |
+| link-checker-api | Checks links on GOV.UK | No change expected |
+| local-links-manager | Manages local links from local authorities on GOV.UK | No change expected |
+| publishing-api | API to publish content on GOV.UK | No change expected |
+| release | Helps deploying to GOV.UK | No change expected |
+| router | Router in front on GOV.UK to proxy to backend servers on the single domain | No change expected |
+| search-admin | Admin for GOV.UK search | No change expected |
+| short-url-manager | Tool to request, approve and create short URL redirects on GOV.UK | No change expected |
+| signon | Single sign-on service for GOV.UK | No change expected |
+| support-api | API for processing GOV.UK named requests and anonymous feedback | No change expected |
+| info-frontend | Serves /info pages to display user needs and performance data about a page on GOV.UK | No change expected |
+| mapit | GOV.UK fork of Mapit, a web service to map postcodes to administrative boundaries | No change expected |
```

Comment:
@sihugh could we long-term use a hosted version of Mapit?

Reply:
There is this: https://mapit.mysociety.org/pricing/. We're just talking about having to do quarterly updates of MapIt data and how much of a pain it is. Leaving this to a service to do for us could save a lot of time and grief.

Reply:
I'll all for using a hosted version. I'm pretty sure we could save £££ even with going for the £300/month option, when you take in to account both existing hosting costs, plus time taken to maintain / update the dataset.

Reply:
Last I looked, there were a couple of small problems:
 - hosted mapIt didn't have official names for councils (might be fixed now)
 - we've added local authority slugs to our mapIt to help with the postcode -> council page journey which would need some unravelling

I'd love a hosted version for government.  I've had at least 5 questions from random service teams about what we use.

Reply:
I'm late to the party on this, with old information, but the latest thinking on mapit is in this doc

https://docs.google.com/document/d/1VyxP5qSZzx5cFL4CQk-k0CKU3Z342VP_fky3D_8pebo/edit?usp=sharing_eil&ts=5c07e833

A hosted service was attractive, but there was a concern about the SLA that Mysociety could offer.